Reemit.Decompiler.Clr/Methods/TinyMethodHeader.cs

```diff
@@ -0,0 +1,9 @@
+    namespace Reemit.Decompiler.Clr.Methods;
+
+public class TinyMethodHeader(uint codeSize) : IMethodHeader
+{
+    public uint CodeSize { get; } = codeSize;
+
+    public static TinyMethodHeader Read(BinaryReader reader) =>
+        new((uint)(reader.ReadByte() & ~((1 << 2) - 1)));
```

Comment:
I found this somewhat confusing. Something like `~0x3` or maybe even `0b11111100` might be a little more readable. And do with need to shift this value to the right two bits?

Reply:
I like your suggestion. And yes, the result should be shifted. Good catch.